lib/simplifyExpression/fractionsSearch/cancelLikeTerms.js

```diff
@@ -288,10 +290,14 @@ function cancelTerms(numerator, denominator) {
     return new CancelOutStatus(numerator, denominator, true);
   }
 
-  // case 4: the numerator is a constant and denominator is a polynomial term
+  // case 4: the numerator is a constant and denominator is a polynomial term that has a coefficient
+  // or is multiplication node
   // e.g. 2 / 4x -> 1 / 2x
+  // e.g. ignore cases like:  2 / a and 2 / x^2
```

Comment:
I'm curious what the error was that was happening - the description being only links to sentry isn't great in github because all us non-Socratic peeps can't see it

can you update the PR name, or comment here (or both) with what the error looked like?

cause ` 2 / a` and `2 / x^2` don't cancel anyways right?